cpu_map_keyme2560.h

```diff
@@ -70,24 +61,36 @@
 #define MOTOR_RESET_DDR                         DDRG
 #define MOTOR_RESET_DDR_PIN                     DDG0
 
+/* Define step pulse output pins. NOTE: All step bit pins must be on the same port. */
+#define STEP_DDR      DDRH
+#define STEP_PORT     PORTH
+#define STEP_PIN      PINH
+#define X_STEP_BIT    PH0
+#define Y_STEP_BIT    PH1
+#define Z_STEP_BIT    PH2
+#define C_STEP_BIT    PH3
+/* All step bits */
+#define STEP_MASK     ((1 << X_STEP_BIT) | (1 << Y_STEP_BIT) | (1 << Z_STEP_BIT) | (1 << C_STEP_BIT))
+
```

Comment:
The STEP and DIRECTION defines logically go together. In https://github.com/keyme/grbl/pull/112, I put the MOTOR_RESET defines between STEP and DIRECTION. This is a bit easier to follow IMO.